node/core/pvf/src/worker_common.rs

```diff
@@ -254,9 +258,6 @@ pub async fn cpu_time_monitor_loop(
 					err
 				);
 			}
-
-			// Kill the process.
-			std::process::exit(1);
```

Comment:
What will happen in case of the timeout is that you will go onto the next iteration of `loop`, hit the `err` branch and return from this function. Is this behaviour desired?

Reply:
Good catch!